src/python/PSetTweaks/WMTweak.py

```diff
@@ -570,7 +567,7 @@ def readAdValues(attrs, adname, castInt=False):
                         except ValueError:
                             print("Error parsing %s's %s value: %s", (adname, attr, val))
                     else:
-                        retval[attr] = val
+                        retval[attr] = val.strip('"')
```

Comment:
Not necessarily correct here.  Would advise to move this to the call site (`getSingleScramArch`).